sklearn/datasets/tests/data/toy_quotes.arff

```diff
@@ -0,0 +1,13 @@
+@attribute 'cat_single_quote' {'A', 'B', 'C'}
+@attribute 'cat_double_quote' {"A", "B", "C"}
+@attribute 'cat_without_quote' {A, B, C}
+@attribute 'str_single_quote' string
+@attribute 'str_double_quote' string
+@attribute 'str_without_quote' string
+@attribute 'str_nested_quote_1' string
+@attribute 'str_nested_quote_2' string
+@attribute 'str_internal_quote' string
+@attribute 'class' numeric
+
+@data
+'A',"A",A,'some text',"some text",some text,'"expect double quotes"',"'expect no quotes'",internal' quote,0
```

Comment:
It would be even more interesting as:

```suggestion
'A',"A",A,'some text',"some text",some text,'"expect double quotes"',"'expect no quotes'",'internal' quote,0
```